chat/SettingsView.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <modal
-    :action="l('settings.action')"
+    :action="l('settings.character')"
```

Comment:
Visual nit I just noticed in the beta build: Currently this makes the "Change Settings" confirmation button read "Character Settings" instead, think this should remain .action (or have a new localization for "Save Character Settings")

Reply:
<img width="240" height="54" alt="image" src="https://github.com/user-attachments/assets/9c82aad8-ae09-4c38-ae47-189c263fa892" />